test/ctia/entity/casebook_test.clj

```diff
@@ -42,13 +42,11 @@
 
```

Comment:
In this ns I simply fixed a fixture with new malware schema, factorized this fixture, and removed dead code.